apps/frontend/src/libs/components/app/app.tsx

```diff
@@ -1,5 +1,6 @@
-import reactLogo from "~/assets/img/react.svg";
-import { Link, RouterOutlet } from "~/libs/components/components.js";
+// import reactLogo from "~/assets/img/react.svg";
+// import { Link, RouterOutlet } from "~/libs/components/components.js";
```

Comment:
Remove comment before commiting and creating a PR

Reply:
sorry about this, i have plans to try this code again.